.github/workflows/linux-pr-tests.yml

```diff
@@ -64,4 +59,50 @@ jobs:
 
       - run: brew test-bot --only-cleanup-before
 
-      - run: brew test-bot --only-formulae --build-from-source
+      - name: Run brew test-bot --only-formulae
+         run: |
+           mkdir ~/bottles
+           cd ~/bottles
+           brew test-bot --only-formulae
+
+      - name: Output brew test-bot --only-formulae failures
+        if: always()
+        run: |
+          cat ~/bottles/steps_output.txt
+          rm ~/bottles/steps_output.txt
+
+      - name: Upload logs
+        if: failure()
+        uses: actions/upload-artifact@main
+        with:
+          name: logs (Linux)
+          path: ~/bottles/logs
+
+      - name: Delete logs and home
+        if: always()
+        run: |
+          rm -rvf ~/bottles/logs
+          rm -rvf ~/bottles/home
+
+      - name: Count bottles
+        id: bottles
+        if: always()
+        run: |
+          cd ~/bottles
+          count=$(ls *.json | wc -l | xargs echo -n)
+          echo "$count bottles"
+          echo "::set-output name=count::$count"
+
```

Comment:
Not doing this here?

https://github.com/Homebrew/homebrew-core/blob/7ec2d116a180efd718f0db47e087cf0fe5aa2302/.github/workflows/tests.yml#L150-L161

Reply:
Done. I aligned the code with the one used in the Mac workflow. Thanks.